asciimatics/widgets.py

```diff
@@ -2453,7 +2453,7 @@ class _BaseListBox(with_metaclass(ABCMeta, Widget)):
                  "_on_select", "_validator", "_search", "_last_search", "_scroll_bar", "_parser"]
 
     def __init__(self, height, options, titles=None, label=None, name=None, parser=None,
-                 on_change=None, on_select=None, validator=None):
+                 on_change=None, on_select=None, validator=None, space_delimiter=None):
```

Comment:
Do you need this change?  Looks like you only use the spacing delimiter in the child class and so this is unecessary.

Reply:
Indeed you are right. I have updated the code accordingly. Thanks